File: shoebill_plugins/PlayerVehiclePlugin/src/main/java/lt/ltrp/event/PlayerVehicleDestroyEvent.java
package lt.ltrp.event;

import lt.ltrp.event.vehicle.VehicleEvent;
import lt.ltrp.object.PlayerVehicle;

/**
 * @author Bebras
 *         2016.05.31.
 */
public class PlayerVehicleDestroyEvent extends VehicleEvent {


    public PlayerVehicleDestroyEvent(PlayerVehicle vehicle) {
        super(vehicle);
    }

    @Override
    public PlayerVehicle getVehicle() {
        return (PlayerVehicle)super.getVehicle();
    }
}

File: shoebill_plugins/PlayerVehiclePlugin/src/main/java/lt/ltrp/player/vehicle/event/PlayerVehicleDestroyEvent.kt
package lt.ltrp.player.vehicle.event

import lt.ltrp.player.vehicle.`object`.PlayerVehicle
import lt.ltrp.vehicle.event.VehicleEvent

/**
 * @author Bebras
 * *         2016.05.31.
 */
class PlayerVehicleDestroyEvent(override val vehicle: PlayerVehicle) : VehicleEvent(vehicle) {

}
